client/my-sites/plugins/marketplace/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { navigation, siteSelection } from 'calypso/my-sites/controller';
+import config from '@automattic/calypso-config';
+import {
+	renderDomainsPage,
+	renderMarketplacePlugin,
+	renderMarketplaceTestPage,
+	renderMarketplaceThankYou,
+	renderPluginsSetupStatusPage,
+} from './controller';
+import { makeLayout, render as clientRender } from 'calypso/controller';
+
+export default function () {
+	if ( config.isEnabled( 'marketplace-yoast' ) ) {
```

Comment:
It's worth thinking about what happens in production if someone wants to access `/marketplace` and this is disabled. Perhaps we should just redirect to `wordpress.com`?

Reply:
Hmmm I guess so but we can pick this up later 👍 I will tag it as an issue.

Reply:
➕  on adding a default case handler with a redirection here to avoid broken paths later on. I recommend adding it here and now to avoid the need to track yet another TODO. I'd rather see us implement a short-term solution in code and then iterate on that simple redirect in the future.

Reply:
Agreed it's a rather minor change. You can use an asterisk as a catch-all when declaring routes, as it's done here: https://github.com/Automattic/wp-calypso/blob/dac30586cd5253c2bf463635e869a364e24bcdcc/client/my-sites/comments/index.js#L47

Reply:
Hi @tyxla @jaswrks thanks for that nudge, didn't realise it would be this easy :D
Fixed by d03d31c486c6d083d184793edcc2f744ae2603e0